app-backend/batch/src/main/scala/com/azavea/rf/batch/sentinel2/ImportSentinel2.scala

```diff
@@ -247,15 +250,21 @@ case class ImportSentinel2(startDate: LocalDate = LocalDate.now(ZoneOffset.UTC))
   def run: Unit = {
     logger.info("Importing scenes...")
     Users.getUserById(systemUser).flatMap { userOpt =>
-      findScenes(startDate, userOpt.getOrElse {
-        val e = new Exception(s"User $systemUser doesn't exist.")
-        sendError(e)
-        throw e
-      })
+      logger.info(s"Getting scenes for $startDate")
+      val keys = getSentinel2Products(startDate)
+      Future.sequence {
+        sentinel2Config.datasourceIds.map { case (prefix, id) =>
```

Comment:
I wonder if there is a more efficient approach than this. With this setup we search through the full list of that day's Sentinel scenes twice and each pass through we see if the scenes match for `2A` and then `2B`. Instead it seems like we could do something like extract the prefix and do a match like:

```scala
val datasourceID = prefix match {
    case "S2B" => <S2B DATASOURCE>
    case "S2A" => <S2A DATASOURCE>
    case _ => throw Exception(s"Unknown Datasource: $prefix")
}
```